docs/source/plugins.rst

```diff
@@ -203,6 +205,19 @@ plugin:
        2) The ``run_local`` and ``target_help`` functions, with the ``target`` parameter excluded, as shown
        `here <https://github.com/mlflow/mlflow/blob/master/mlflow/deployments/base.py>`_
      - `PluginDeploymentClient <https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin/fake_deployment_plugin.py>`_.
+   * - Plugins for custom mlflow evaluator.
+     - mlflow.model_evaluator
+     - The entry point name (e.g. ``dummy_evaluator``) is the evaluator name which is used in the ``evaluators`` argument of the ``mlflow.evaluate`` API.
+       The entry point value (e.g. ``dummy_evaluator:DummyEvaluator``) specify a class which is subclass of ``mlflow.models.evaluation.ModelEvaluator``,
+       the subclass must implement 2 methods:
+       1) Implement ``can_evaluate`` method: The method accept keyword-only arguments of ``model_type`` and ``evaluator_config``,
+       return True if the evaluator can evaluate the specified model type with specified evaluator config. False otherwise.
```

Comment:
```suggestion
       1) ``can_evaluate``: Accepts the keyword-only arguments ``model_type`` and ``evaluator_config``.
       Returns ``True`` if the evaluator can evaluate the specified model type with the specified evaluator config. Returns ``False`` otherwise.
```